tests/e2e/specs/add-paid-campaigns/add-paid-campaigns.test.js

```diff
@@ -280,11 +279,15 @@ test.describe( 'Set up Ads account', () => {
 	} );
 
 	test.describe( 'Create your paid campaign', () => {
+		const launchPaidCampaignButton = page.getByRole( 'button', {
```

Comment:
@kt-12 Let's move this within a function in `tests/e2e/utils/pages/setup-ads/setup-budget.js`

Reply:
Move as a function in `setup-budget.js`